src/components/FeaturedProducts/FeaturedProducts.js

```diff
@@ -25,18 +22,53 @@ export default function FeaturedProducts(props) {
                     alt: data.alt,
                     text: data.text,
                     navigationLink: data.navigationLink,
-                    uniqueId: data.uniqueId
+                    uniqueId: data.uniqueId,
+                    stock:data.stock,
+                    name:data.name, 
+                    price:data.price, 
                 };
             })
         });
-    // console.log(featuredProducts)  
+
+
+
+    function addItemToCard(item) {
```

Comment:
This code should in the Context and you should use `useReducer`, right now you should repeat this code in every place where you add a product.